src/Arcus.Testing.Assert/AssertCsv.cs

```diff
@@ -74,6 +75,21 @@ public AssertCsvOptions IgnoreColumn(string headerName)
         /// </summary>
         internal IReadOnlyCollection<string> IgnoredColumns => _ignoredColumns;
 
+        /// <summary>
+        /// Adds a column via a zero-based index which will get ignored when comparing CSV tables.
+        /// </summary>
+        /// <param name="index">The zero-based index of the column that should be ignored.</param>
+        public AssertCsvOptions IgnoreColumnIndex(int index)
```

Comment:
Maybe also provide a guard that throws an `ArgumentOutOfRangeException` when the `index` is negative.

Reply:
Fixed in [3db9daf](https://github.com/arcus-azure/arcus.testing/pull/161/commits/3db9dafaa8297dd3aabc81f97a80796c3c968eb0)